readme.md

```diff
@@ -20,6 +20,11 @@ $ npm install pretty-time-zsh
 $ antigen bundle sindresorhus/pretty-time-zsh
 ```
 
+### Install with Zgen
+
+1. Add `zgen load sindresorhus/pretty-time-zsh` to your `.zshrc` with your other `zgen load` statements
+2. `zgen reset`
+3. Open a new shell and zgen will clone pretty-time-zsh and regenerate init.zsh
```

Comment:
This is too much info. I'm not interested in documenting how Zgen works. That's better done in the Zgen readme.

This should be more than enough:

> Add `zgen load sindresorhus/pretty-time-zsh` to your `.zshrc` and run `$ zgen reset`.